qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -580,7 +597,7 @@ def clear_buffers(self) -> None:
         self.log.debug("buffers cleared")
         self.buffer_list = []
 
-    def signal_to_volt(self, channel: int, signal: int) -> float:
+    def signal_to_volt(self, channel: int, signal: float) -> float:
```

Comment:
just double checking if signal is really a float - "convert a value from a buffer" might mean that the value is actually integer, because alazar "measures" integers, and then we convert them to float volts, but i don't remember for sure.

it does not mean though that this method can't be called on integer-values floats or any floats, but i think it was meant to be called on integers that alazar puts into the buffer.

Reply:
I changed this to float since we are actually passing a float to this method from `Demodulation_AcquisitionController` Note that the center point will always be a float (127.5) which is the reason for this issue

Reply:
oh... ok :)